packages/button/src/ButtonIcon.tsx

```diff
@@ -90,6 +91,13 @@ const onlyIconColor: Record<Mode, Record<Variant, string>> = {
   },
 };
 
+const onlyIconStyleHover = css`
+  ${ButtonDataProp.selector}:hover &,
```

Comment:
Out of curiosity, is this a way to target the parent?

Reply:
Unfortunately, yes. The other way to do it would be by switching a css variable in the parent styles